storage/migrate.go

```diff
@@ -94,6 +94,12 @@ func (d *Database) MigrateSchemaTo(ctx context.Context, target int) error {
 		return xerrors.Errorf("acquiring schema lock: %w", err)
 	}
 
+	// CREATE EXTENSION cannot run inside a transaction (as part of
```

Comment:
We only need to do this when we are at schema version 0 (when dbVersion = 0)

Reply:
Sorry for the delay, added that now.